katib/katib-controller/base/katib-mysql-pv.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```

Comment:
Is this needed?

Reply:
PV and pvc changes are not required

Reply:
I think it is useful when you are not using GCP and you don't have StorageClass on your cluster. We have this PV in Katib manifest.
What do you think @richardsliu @johnugeorge ?

Reply:
But then, need not be in base in that case.

Reply:
@johnugeorge @richardsliu Ok, I removed it from base.